docs/pipeline/sources/update.mdx

```diff
@@ -33,33 +44,11 @@ To undo the override, simply erase the value from the `service.name` field.
 
 ### Kubernetes Manifest
 
-If you prefer to use kubectl or GitOps to manage your Kubernetes workloads, you can can set the `service.name` attribute by adding an annotation to the workload manifest.
-
-For example, to set the `service.name` for a deployment named `my-deployment`, to a value of `foo` with kubectl, you can run:
+If you prefer to use kubectl or GitOps to manage your Kubernetes workloads, you can can set the `service.name` attribute by updating the corresponding Source.
 
-```shell
-kubectl annotate deployment my-deployment odigos.io/reported-name=foo --overwrite
-```
-
-To view the changes, run:
+For example, to set the `service.name` attribute in telemetry for an existing source:
 
 ```shell
-kubectl get deployment my-deployment -o yaml
+kubectl patch source/<source-name> -n <namespace> --type=merge -p '{"spec":{"otelServiceName":"<serviceName>"}}'
```

Comment:
Users probably know if it's deployment / daemonset / statefulset, it's name, and it's namespace.
Do you have the `kubectl` command that translate user properties to relevant source? could be so awesome to have it here as reference and better usability.